Pyrado/pyrado/environments/rcspysim/box_flipping.py

```diff
@@ -44,8 +44,8 @@
 from pyrado.utils.data_types import EnvSpec
 
 
-rcsenv.addResourcePath(rcsenv.RCSPYSIM_CONFIG_PATH)
-rcsenv.addResourcePath(osp.join(rcsenv.RCSPYSIM_CONFIG_PATH, "BoxFlipping"))
+rcsenv.addResourcePath(rcsenv.RCSPYSIM_CONFIG_PATH)  # pylint: disable=no-member
```

Comment:
why is that? the module `rcsenv` exposes a function called `addResourcePath()` :thinking:

Reply:
If the members are dynamically generated or imported with C binds or so, PyLint does not understand that they are there. It's the same for NumPy and PyTorch. However, I excluded them globally from member-checking and forgot about that possibility when going through the other issues. I added it for rcsenv too, but forgot to remove the ignores everywhere. Will do ;)